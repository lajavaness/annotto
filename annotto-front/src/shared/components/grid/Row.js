import PropTypes from 'prop-types'
import React from 'react'

import * as Styled from './__styles__/Row.styles'

const Row = (props) => {
	const { children, ...otherProps } = props
	return <Styled.Root {...otherProps}>{children}</Styled.Root>
}

export default Row

Row.propTypes = {
	/** Defines the height of the row. */
	height: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
	/** Defines the title of the card. */
	children: PropTypes.node,
}

Row.defaultProps = {
	height: null,
	children: null,
}
