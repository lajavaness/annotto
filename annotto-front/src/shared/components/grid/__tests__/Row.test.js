import { render } from '@testing-library/react'
import React from 'react'

import Row from 'shared/components/grid/Row'

describe('Row', () => {
	it('matches snapshot', () => {
		const { asFragment } = render(<Row>Foo</Row>)
		expect(asFragment()).toMatchSnapshot()
	})
})
