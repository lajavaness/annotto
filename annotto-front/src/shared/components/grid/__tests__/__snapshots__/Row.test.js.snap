// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="ant-row sc-bdfBQB gaChjQ"
  >
    Foo
  </div>
</DocumentFragment>
`;
